data/common.yaml

```diff
@@ -0,0 +1,2 @@
+# ---
+# hiera_data: "special data"
```

Comment:
This key name is kind of meta and makes it read like this is a config file for hiera. We should use something more generic, and probably include a comment explaining the structure of the file